File: LC75.cpp
class Solution {
public:
    void sortColors(vector<int>& nums) {
        int v[3] = {0,0,0};
        

        for(auto i : nums)
            v[i]++;
        
        int i = 0, j = 0;

        while(i < nums.size()){
            while(v[j]){
                nums[i++] = j;
                v[j]--;
            }
            j++;
        }

        //return nums;
    }
};

File: LC75.py
class Solution:
    def sortColors(self, nums: List[int]) -> None:
        """
        Do not return anything, modify nums in-place instead.
        """
        l = [0]*3

        for i in nums:
            l[i] += 1
        
        j = 0; i = 0
        while i < len(nums):
            
            while(l[j]):
                nums[i] = j
                i += 1
                l[j] -= 1
            
            j += 1
        
